julia.cmake

```diff
@@ -0,0 +1,15 @@
+# .rst: .. command:: JULIA_CHECK_PACKAGE (PKG_NAME)
+#
+# Test if package PKG_NAME is installed in Julia.
+#
+macro(JULIA_CHECK_PACKAGE PKG_NAME)
+  set(file_content
+      "try\n@eval import ${PKG_NAME}\nprintln(1)\ncatch\nprintln(0)\nend")
+  file(WRITE ${PROJECT_SOURCE_DIR}/_tmp_cmake_julia/test_julia_package.jl
+       ${file_content})
+  execute_process(
+    COMMAND ${Julia_EXECUTABLE}
+            ${PROJECT_SOURCE_DIR}/_tmp_cmake_julia/test_julia_package.jl
+    OUTPUT_VARIABLE ${PKG_NAME}_found)
```

Comment:
Maybe to avoid any clash, we should rename `${PKG_NAME}_found` into `Julia_${PKG_NAME}_found` and add the output variable in the doc.